### ISSUE TITLE
RFE: wwctl overlay touch

### ISSUE THREAD

**Issue by panda1100** [hpcng/warewulf#858]
This RFE propose adding `touch` command to `wwctl overlay`.

I recently found that when we want to create new file on overlay (The file does not exist on host, so import does not work scenario), it's not easily write a script for automation due to it's interactive editor.

`wwctl overlay touch` add completely new file to overlay non-interactively.

```
sudo wwctl overlay touch --parents generic /etc/security/access.conf
```

with above pseudo command I can write something like this way in script.

```
cat << EOF | sudo tee -a $ROCKY9/etc/security/access.conf
+ : root : ALL
+ : ssh-allowed : ALL
- : ALL : ALL
EOF
```

Currently  when I add access.conf to overlay, I use `wwctl overlay edit`.

```
sudo wwctl overlay edit --parents generic /etc/security/access.conf
```

and manually add lines to the file.

```
+ : root : ALL
+ : ssh-allowed : ALL
- : ALL : ALL
```

**Comment by anderbubble** [comment]
`touch` implies more than just creating files. This leaves me wanting to have a general-purpose `wwctl overlay exec` or similar, rather than just re-implementing all unix file management utilities within `wwctl overlay` command. Thoughts?

**Comment by anderbubble** [comment]
@panda1100 how does a `touch` command enable `$ROCKY9/etc/security/access.conf` in your use case?

**Comment by panda1100** [comment]
@anderbubble The following command is kind of annoying when we write a document.

This was while ago, and maybe I'm wrong but as far as I understand,,
to add new file to overlay, 

(1) create file with the following command

```
sudo wwctl overlay edit --parents generic /etc/security/access.conf
```

(2) and then edit manually with the following content

```
+ : root : ALL
+ : ssh-allowed : ALL
- : ALL : ALL
```

I prefer something that enable us to do copy-pastable way of file creation on overlay.

**Comment by griznog** [comment]
> I prefer something that enable us to do copy-pastable way of file creation on overlay.

```
cat > /path/to/overlays/generic/etc/security/access.conf << EOF
+ : root : ALL
+ : ssh-allowed : ALL
- : ALL : ALL
EOF
```
or, using an editor to literally copy/paste:
```
EDITOR=vim # Or whatever you like to use
$EDITOR /path/to/overlays/generic/etc/security/access.conf
# copy/paste as desired and write the file
```

I'm convinced wrapping normal filesystem functions with `wwctl` causes more harm than good by implying there is something magical going on behind the curtain, although the distraction of putting any effort at all into maintaining them in `wwctl` is also pretty harmful.

**Comment by anderbubble** [comment]
> I prefer something that enable us to do copy-pastable way of file creation on overlay.

I agree; but the proposed `touch` doesn't help you get the value for `$ROCKY9`.

> I'm convinced wrapping normal filesystem functions with wwctl causes more harm than good

Yeah, I'd like to work towards unwinding this. See #488.
